api/nodes/serializers.py

```diff
@@ -319,10 +334,10 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
         related_meta={'count': 'get_registration_count'}
     ))
 
-    region = RelationshipField(
+    region = RegionRelationshipField(
         related_view='regions:region-detail',
-        related_view_kwargs={'region_id': '<osfstorage_region._id>'},
-        read_only=True
+        related_view_kwargs={'region_id': 'get_region_id'},
```

Comment:
Looks like I wasn't that clever here. Region can't be embedded and isn't serialized *quite* right:
![screen shot 2018-06-11 at 4 50 49 pm](https://user-images.githubusercontent.com/7913604/41256525-d854b268-6d97-11e8-8fd5-30dc0921293e.png)
![screen shot 2018-06-11 at 4 51 35 pm](https://user-images.githubusercontent.com/7913604/41256524-d842c5e4-6d97-11e8-9e0c-d64c696c0bf1.png)

Reply:
Ah, yeah, you can only get attributes...maybe we should change that to allow method names? I can't think of a use case for passing a hard-coded value, e.g. `related_view_kwargs={'region_id': 'us-east'}`.

Reply:
Yeah... I could pass the property here, but then we lose all of the annotation goodness. 
`related_view_kwargs={'region_id': 'osfstorage_region'}`